test/integration-tests/UniswapV4.test.ts

```diff
@@ -37,17 +37,35 @@ describe('Uniswap V4 Tests:', () => {
   let v4PoolManager: PoolManager
   let v4PositionManager: PositionManager
 
+  // current market ETH price at block
+  const USD_ETH_PRICE = 3820
+  const ONE_PERCENT = 38
```

Comment:
nit same comment on another PR but I didnt initially realize the ONE_PERCENT var meant 1% OF USD_ETH_PRICE